composer.json

```diff
@@ -67,5 +68,10 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "scripts": {
```

Comment:
> Still, I'm using directly api-platform/core, so it is my root :p

And of course it's not going to work for any actual projects *using* API Platform.

> I'd add that copying isn't the best solution in term of space needed as vendor files are then useless and take 5.4M for nothing.

That's already the case for bundle resources. I don't think it's a concern since the user is doing it for good reasons. The default should be relative symlink, just like in Symfony.